dataland-frontend/src/components/generics/ViewMultipleDatasetsDisplayBase.vue

```diff
@@ -0,0 +1,285 @@
+          No {{ humanizedDataDescription }} data could be found for the data ID passed in the URL for this company and
+          framework.
+        </h2>
+      </div>
+      <div v-if="isReportingPeriodInUrlInvalid" data-test="noDataForThisReportingPeriodPresentErrorIndicator">
+        <h2>
+          No {{ humanizedDataDescription }} data could be found for the reporting period passed in the URL for this
+          company.
+        </h2>
+      </div>
+    </template>
+  </ViewFrameworkBase>
+</template>
+
+<script lang="ts">
+import ViewFrameworkBase from "@/components/generics/ViewFrameworkBase.vue";
+import { defineComponent, inject } from "vue";
+import { DataMetaInformation } from "@clients/backend";
+import { humanizeString } from "@/utils/StringHumanizer";
+import LksgPanel from "@/components/resources/frameworkDataSearch/lksg/LksgPanel.vue";
+import { DataTypeEnum } from "@clients/backend";
+import SfdrPanel from "@/components/resources/frameworkDataSearch/sfdr/SfdrPanel.vue";
+import { ApiClientProvider } from "@/services/ApiClients";
+import { assertDefined } from "@/utils/TypeScriptUtils";
+import { AxiosError } from "axios";
+import Keycloak from "keycloak-js";
+import DatasetStatusIndicator from "@/components/resources/frameworkDataSearch/DatasetStatusIndicator.vue";
+
+export default defineComponent({
+  name: "ViewMultipleDatasetsDisplayBase",
+  components: { DatasetStatusIndicator, SfdrPanel, LksgPanel, ViewFrameworkBase },
+  props: {
+    companyId: {
+      type: String,
+    },
+    dataType: {
+      type: String,
+    },
+    dataId: {
+      type: String,
+    },
+    reportingPeriod: {
+      type: String,
+    },
+  },
+  data() {
+    return {
+      isWaitingForListOfDataIdsToDisplay: true,
+      isListOfDataIdsToDisplayFound: false,
+      receivedMapOfDistinctReportingPeriodsToActiveDataMetaInfo: {} as Map<string, DataMetaInformation>,
+      singleDataMetaInfoToDisplay: null as null | DataMetaInformation,
+      distinctAvailableReportingPeriods: [] as string[], // TODO note:    this is needed for the popup when you click "EDIT"
+      humanizeString: humanizeString,
+      isDataIdInUrlInvalid: false,
+      isReportingPeriodInUrlInvalid: false,
+      humanizedDataDescription: humanizeString(this.dataType),
+      DataTypeEnum,
+    };
+  },
+  setup() {
+    return {
+      getKeycloakPromise: inject<() => Promise<Keycloak>>("getKeycloakPromise"),
+    };
+  },
+  computed: {
+    linkToActiveView() {
+      const activeDatasetAvailable =
+        this.receivedMapOfDistinctReportingPeriodsToActiveDataMetaInfo &&
+        this.receivedMapOfDistinctReportingPeriodsToActiveDataMetaInfo.size > 0;
+
+      if (this.companyId && this.dataType && activeDatasetAvailable)
+        return `/companies/${this.companyId}/frameworks/${this.dataType}`;
+      return undefined;
+    },
+  },
+  watch: {
+    dataId(newDataId: string) {
+      console.log("dataID watcher in Multi-View component executed"); // TODO
```

Comment:
still console logs here

Reply:
I deleted unnecessary console logs. I kept the ones which are executed in catch blocks.